### PR TITLE
Fix bug in debug temporary input

### PR DESCRIPTION
**Pull request by phabh**
Fixing bug that not appear temporary input in debug.

### CODE REVIEW

**Comment by andreminelli** [comment]
@phabh , ideally every bug found should have a test to 'expose' it and prevent a new problem in the future. Can you add a test for this, please?

File: src/Take.Blip.Builder.UnitTests/InputExpirationHandlerTests.cs [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+            InputHandler = new InputExpirationHandler(Scheduler);
+        }
+
+        public Identity UserIdentity { get; }
+        public Identity ApplicationIdentity { get; }
+        public Application Application { get; }
+        public Message Message { get; }
+
+        private ISchedulerExtension Scheduler;
```
Comment (ceifa) [inline src/Take.Blip.Builder.UnitTests/InputExpirationHandlerTests.cs]: Private fields should appear before constructor

Comment (ceifa) [inline src/Take.Blip.Builder.UnitTests/InputExpirationHandlerTests.cs]: Make this readonly

File: src/Take.Blip.Builder.UnitTests/InputExpirationHandlerTests.cs [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+            var returnMessage = InputHandler.ValidateMessage(Message);
+
+            // Assert
+            Assert.True(returnMessage.Metadata.ContainsKey(TraceSettings.BUILDER_TRACE_TARGET));
+            Assert.True(returnMessage.Metadata.ContainsKey(TraceSettings.BUILDER_TRACE_TARGET_TYPE));
+            Assert.True(returnMessage.Metadata.ContainsKey(TraceSettings.BUILDER_TRACE_MODE));
+            Assert.True(returnMessage.Metadata.ContainsKey("inputExpiration.stateId"));
+            Assert.True(returnMessage.Metadata.ContainsKey("inputExpiration.identity"));
+            Assert.False(Message.Metadata.ContainsKey("inputExpiration.identity"));
+            Assert.False(Message.Metadata.ContainsKey("inputExpiration.stateId"));
```
Comment (ceifa) [inline src/Take.Blip.Builder.UnitTests/InputExpirationHandlerTests.cs]: Make these constants as public on InputExpirationHandler and use it here

**Pull request merged**